src/utils/workspaceUtils.ts

```diff
@@ -136,3 +136,7 @@ export class NoStaticResourcesDirError extends Error {
         Object.setPrototypeOf(this, NoStaticResourcesDirError.prototype);
     }
 }
+
+export function getExtensionName(context: ExtensionContext): string {
```

Comment:
Could get extension be a string separated by space? Will that cause problem?

Reply:
this is the package.json name attribue, currently it's 'salesforcedx-vscode-mobile'. not like to change it to have space.